rag_experiment_accelerator/evaluation/plain_metrics.py

```diff
@@ -1,153 +1,163 @@
 import evaluate
-import textdistance
-from fuzzywuzzy import fuzz
-
-algorithms = textdistance.algorithms
+from rapidfuzz import fuzz
+from rapidfuzz import distance as d
+from textdistance import algorithms as alg
 
 
 # https://huggingface.co/spaces/evaluate-metric/bleu
-def bleu(predictions, references):
-    bleu = evaluate.load("bleu")
-
-    results = bleu.compute(predictions=predictions, references=references, max_order=2)
-    # multiplying by 100 to maintain consistency with previous implementation
-    return results["bleu"] * 100
-
-
-def fuzzy(doc1, doc2):
+def bleu(predictions: list[str], references: list[str]) -> float:
     """
```

Comment:
Severity: Major - The type hints for the `predictions` and `references` parameters are incorrect. They should be `list[str]` instead of `list`.


[Hack - peer-review-pal] We are sorry for the intrusion. Feel free to ignore/delete the comments with the tag - they are automatically generated by [our Hack project](https://github.com/microsoft/peer-review-pal).
If you find them useful, want us to delete them or have any feedback or want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding #prp-ignore to your PR description.